src/components/Link/Link.tsx

```diff
@@ -15,6 +16,176 @@ interface LinkProps extends AnchorHTMLAttributes<HTMLAnchorElement> {
   underline?: boolean;
 }
 
-export function Link({ children }: PropsWithChildren<LinkProps>) {
-  return <>{children}</>;
+/**
+ * - `underline` 속성으로 밑줄 표시 여부를 설정할 수 있습니다. 기본값은 `true`입니다.
+ * - `target="_blank"`를 사용할 때는 자동으로 보안 속성이 추가됩니다.
+ * - `tone` 속성을 통해서 링크의 색상을 변경할 수 있습니다. 기본값은 `neutral`입니다.
+ * - `size` 속성과 `weight` 속성을 통해서 텍스트 스타일을 변경할 수 있습니다.
+ * - `muted` 속성을 주시면 글자색이 옅어집니다. 명암비가 낮아지므로 접근성 측면에서 주의해서 사용하세요. 기본값은 `false`입니다.
+ */
+export function Link({
+  children,
+  tone = "neutral",
+  size,
+  weight,
+  muted = false,
+  underline = true,
+  target,
+  rel,
+  onClick,
+  ...props
+}: PropsWithChildren<LinkProps>) {
+  const handleKeyDown = (e: React.KeyboardEvent<HTMLAnchorElement>) => {
+    if (e.key === "Enter" && props.href) {
+      if (onClick) {
+        onClick({} as React.MouseEvent<HTMLAnchorElement>);
+      } else {
+        window.location.href = props.href;
+      }
+    }
+  };
+
+  const handleClick = (e: React.MouseEvent<HTMLAnchorElement>) => {
+    if (onClick) onClick(e);
```

Comment:
nitpick: 이렇게 작성할 수도 있습니다!
```suggestion
    onClick?.(e);
```

Reply:
옵셔널체이닝 방식이 더 간결하네요 감사합니다.